src/walletSdk/Horizon/Transaction/TransactionBuilder.ts

```diff
@@ -90,6 +90,35 @@ export class TransactionBuilder extends CommonTransactionBuilder<TransactionBuil
     return this;
   }
 
+  pathPay(
+    destinationAddress: string,
+    sendAsset: StellarAssetId,
+    destAsset: StellarAssetId,
+    amount: string,
+    destMin?: string,
+  ): TransactionBuilder {
+    this.operations.push(
+      StellarSdk.Operation.pathPaymentStrictSend({
```

Comment:
kind of similar to `destMin`. If I just want simple case of sending x amount of 1 currency and have receiver receive different currency I don't really care about whether it's `strictReceive` or `strictSend`

I guess it depends on the target audience for this sdk, with have stellar-savy they are

imo I think we should only add a wrapper for strictSend to simplify, and if a developer wants to do strictReceive they can do using `addOperation`

Reply:
I'm happy to add both methods, lmk if you guys disagree with my thinking
cc @Ifropc @piyalbasu

Reply:
maybe an optional param flag to `pathPay` to decide between `strictSend` or `strictReceive` is best

Reply:
maybe instead of amount add optional amountSend and amountReceive 
1. Check only one of them specified
2. Do send/receive depending on that amount is provided 
WDYT?

Reply:
ya I like that plan 👍

Reply:
updated https://github.com/stellar/typescript-wallet-sdk/pull/56/commits/a897a3fff809af244ff5bdb489de65d898c67a74